src/main/java/net/neoforged/neoforge/common/BooleanAttribute.java

```diff
@@ -34,4 +40,32 @@ public double sanitizeValue(double value) {
         }
         return value > 0 ? 1 : 0;
     }
+
+    @Override
+    public MutableComponent toValueComponent(@Nullable Operation op, double value, TooltipFlag flag) {
+        if (op == null) {
+            return Component.translatable("neoforge.value.boolean." + (value > 0 ? "enabled" : "disabled"));
+        } else if (op == Operation.ADD_VALUE && value > 0) {
+            return Component.translatable("neoforge.value.boolean.enable");
+        } else if (op == Operation.ADD_MULTIPLIED_TOTAL && (int) value == -1) {
+            return Component.translatable("neoforge.value.boolean.force_disable");
+        } else {
+            return Component.translatable("neoforge.value.boolean.invalid");
+        }
+    }
+
+    @Override
+    public MutableComponent toComponent(AttributeModifier modif, TooltipFlag flag) {
+        double value = modif.amount();
+
+        MutableComponent comp;
+        if (value > 0.0D) {
+            comp = Component.translatable("neoforge.modifier.bool", this.toValueComponent(modif.operation(), value, flag), Component.translatable(this.getDescriptionId())).withStyle(ChatFormatting.BLUE);
+        } else {
+            value *= -1.0D;
+            comp = Component.translatable("neoforge.modifier.bool", this.toValueComponent(modif.operation(), value, flag), Component.translatable(this.getDescriptionId())).withStyle(ChatFormatting.RED);
```

Comment:
Instead of copying this long line twice, can we just set a `ChatFormatting` variable instead of one for `MutableComponent`?